code/ship/ship.cpp

```diff
@@ -4707,31 +4696,28 @@ static void parse_ship_values(ship_info* sip, const bool is_template, const bool
 		}
 	}
 
-	man_thruster *mtp = NULL;
-	man_thruster manwich;
 	while(optional_string("$Thruster:"))
 	{
+		rcs_thruster_info *mtp = nullptr;
+		rcs_thruster_info manwich;
```

Comment:
These should probably be renamed, as funny as `manwich` was, it's a bit of a non-sequitur now. `rtp` and `rcs_info` or something

Reply:
And nothing is stopping us from putting a comment, too `rcs_trhuster_info rcs_info;  // The variable formerly known as manwich`

Reply:
Fair enough